src/app/examples/nativescript-only/list-section-example/list-section-example.component.tns.ts

```diff
@@ -1,13 +1,20 @@
 import { Component } from '@angular/core';
 
-import { BaseListComponent } from '../../base-list.component';
+import { BaseListComponent } from '~/app/examples/list/base-list.component';
```

Comment:
Don't use absolute imports - it's easer to grasp that this lives inside the examples folder as does this component

Reply:
Fixed!

Reply:
Reverted back to relative import